recipes/upstream/azure/rke/main.tf

```diff
@@ -0,0 +1,64 @@
+locals {
+  local_ssh_private_key_path = var.ssh_private_key_path == null ? "${path.cwd}/${var.prefix}-ssh_private_key.pem" : var.ssh_private_key_path
+  local_ssh_public_key_path  = var.ssh_public_key_path == null ? "${path.cwd}/${var.prefix}-ssh_public_key.pem" : var.ssh_public_key_path
+  startup_script             = var.os_type == "sles" ? var.startup_script : "#!/bin/bash\nexport DEBIAN_FRONTEND=noninteractive ; curl -sSL https://releases.rancher.com/install-docker/20.10.sh | sh - ; sudo usermod -aG docker ubuntu ; newgrp docker ; sudo sysctl -w net.bridge.bridge-nf-call-iptables=1 ; sleep 180"
```

Comment:
This serves the purpose but would be nice to handle this logic in a shell script and allow the docker version to be set